src/couch/src/couch_util.erl

```diff
@@ -870,3 +871,19 @@ remove_sensitive_data(KVList) ->
     KVList1 = lists:keyreplace(<<"password">>, 1, KVList, {<<"password">>, <<"****">>}),
     % some KVList entries are atoms, so test fo this too
     lists:keyreplace(password, 1, KVList1, {password, <<"****">>}).
+
+% Alternative to `lists:uniq/1` until OTP 26 becomes the oldest
+% supported version.
+-spec nub(list()) -> list().
+nub(L) ->
+    nub_int(L, sets:new()).
```

Comment:
Since we support OTP 24+ we can use a bit of a faster version of sets with `sets:new([{version, 2}]).`

https://www.erlang.org/doc/man/sets.html

It might not matter for indexable_fields as much but since it's in `couch_util` we might end up using some place where it might make a difference.

Reply:
Yes, I agree.  Thanks for the comment!